docs/content/manual/backup-and-restore/4954-cannot-resotre-new-vm-have-snapshot.md

```diff
@@ -0,0 +1,27 @@
+---
+title: VM have snapshot can't be restored to new VM
+---
+
+* Related issues: [#4954](https://github.com/harvester/harvester/issues/4954) [BUG] VM Restore to new VM doesn't work if there is VM Snapshot
+
+
+## Category: 
+* Backup and Restore
+
+## Verification Steps
+1. Prepare a 3 nodes Harvester cluster
+1. Create a vm named vm1
+1. Setup nfs backup target
+1. Create a backup for the vm1
+1. Create a snapshot for the vm1
+1. Restore the backup of vm1 to create a new VM
+1. Check can restore vm correctly
+1. Shutdown vm1,
```

Comment:
should not end with comma.

Reply:
Thanks for the check.
Remove the comma in line 19.